clinics/cvs.py

```diff
@@ -29,7 +29,7 @@ def get_locations(self):
             },
             "requestPayloadData": {
                 "selectedImmunization": ["CVD"],
-                "distanceInMiles": 35,
+                "distanceInMiles": os.environ["RADIUS"],
```

Comment:
Actually let's add the int cast back here - I was thinking url params where type wouldn't matter, let's stick with their type
```suggestion
                "distanceInMiles": int(os.environ["RADIUS"]),
```